core/rollup_l1_cost.go

```diff
@@ -20,17 +20,15 @@ import (
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/params"
 )
 
-var big10 = big.NewInt(10)
-
 var (
 	L1BaseFeeSlot = common.BigToHash(big.NewInt(1))
-	OverheadSlot  = common.BigToHash(big.NewInt(3))
-	ScalarSlot    = common.BigToHash(big.NewInt(4))
-	DecimalsSlot  = common.BigToHash(big.NewInt(5))
+	OverheadSlot  = common.BigToHash(big.NewInt(5))
+	ScalarSlot    = common.BigToHash(big.NewInt(6))
 )
 
 var (
```

Comment:
Is `OVM_GasPriceOracleAddr` used anymore?

Reply:
It's not, I'll remove it